### ISSUE TITLE
Mask image of InterHand2.6M

### ISSUE THREAD

**Issue by redorangeyellowy** [zijinxuxu/SMHR#1]
Thank you for the good research.

I want to get a mask image for InterHand2.6M, but I wonder if there is a pre-trained file trained with this dataset.

**Comment by zijinxuxu** [comment]
Since the InterHand2.6M dataset does not provide mask data, we remove the loss of the rendering part during training. In fact, mask is not necessary in our weakly-supervised framework. Our model is designed to deal with general multi handed situations, so we only select a small part of InterHand 2.6M data to verify the feasibility of the method.
If you need to use mask for training, I suggest using the "InterHand2.6M_train_MANO_NeuralAnnotat.json" file provided by the dataset to render by yourself.

**Comment by redorangeyellowy** [comment]
Okay. Thank you for your reply.
